src/pybind/mgr/dashboard/frontend/src/app/app-routing.module.ts

```diff
@@ -179,15 +179,15 @@ const routes: Routes = [
     canActivate: [AuthGuardService],
     canActivateChild: [AuthGuardService],
     data: { breadcrumbs: 'Pools' },
-    loadChildren: './ceph/pool/pool.module#RoutedPoolModule'
+    loadChildren: () => import('./ceph/pool/pool.module').then((m) => m.RoutedPoolModule)
```

Comment:
Interesting change. More verbose, but if it [helps catch typos before build time](https://www.youtube.com/watch?v=jPXl7sCPCOA), I'm happy with that.